Java/build.ci.cmd

```diff
@@ -0,0 +1,17 @@
+@ECHO off
+
+ECHO # Java environment info
+CALL mvn -v
+
+ECHO # Clean resources
+DEL /S /Q libraries\recognizers-text-number\src\main\java\com\microsoft\recognizers\text\number\resources\*.java
```

Comment:
Can't we just call here the same resource generation script that already exists?

Reply:
This would avoid two build scripts.